### ISSUE TITLE
main branch contents

### ISSUE THREAD

**Issue by Phillip-D-Shields** [nisa-nz/resources#5]
does the main branch need to keep the resources in it if we deploy from gh-pages branch? could we just have a readme in main, deploy build on gh-pages, and dev stuff on dev branches?

@MCKevmeister

**Comment by MCKevmeister** [comment]
Yes, I this would work. I am unsure on best practice at this stage.

**Comment by MCKevmeister** [comment]
Suggest closing this issue
